reference/3.0/Microsoft.PowerShell.Utility/Compare-Object.md

```diff
@@ -223,20 +229,27 @@ Accept wildcard characters: False
 
 ### CommonParameters
 
-This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see about_CommonParameters (http://go.microsoft.com/fwlink/?LinkID=113216).
+This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](http://go.microsoft.com/fwlink/?LinkID=113216).
 
 ## INPUTS
 
 ### System.Management.Automation.PSObject
 
-You can pipe a DifferenceObject object to Compare-Object.
+You can pipe a **DifferenceObject** object to this cmdlet.
 
 ## OUTPUTS
 
-### None, or the objects that are different
+### None
+
+If the objects are the same, nothing is returned.
+
+### System.Management.Automation.PSCustomObject
 
-When you use the PassThru parameter, Compare-Object returns the objects that differed.
-Otherwise, this cmdlet does not generate any output.
+If the objects are different, `Compare-Object` wraps the differing objects in a `PSCustomObject` wrapper with a `SideIndicator` property to reference the differences.
+
+### Other
```

Comment:
Get rid of this H3. The next paragraph should be merged with the paragraph above.